_data/meltano/loaders/target-clickhouse/shaped.yml

```diff
@@ -0,0 +1,50 @@
+capabilities:
```

Comment:
@BTheunissen this file should be named `shaped-ai.yml` to match the variant name.